src/plugins/orangehrmDashboardPlugin/config/routes.yaml

```
@@ -12,6 +12,13 @@ apiv2_dashboard_employees_by_subunit:
   defaults:
     _api: OrangeHRM\Dashboard\Api\EmployeeDistributionBySubunitAPI
 
+apiv2_dashboard_quick_launch:
+  path: /api/v2/dashboard/quick-launch
```

```suggestion
  path: /api/v2/dashboard/shortcuts
```
:thinking: this also fine

---

yes this is better changed it